### ISSUE TITLE
`import-notation` false positive with Less

### ISSUE THREAD

**Issue by SevenOutman** [stylelint/stylelint#7096]
### What minimal example or steps are needed to reproduce the bug?

I've create a [repository](https://github.com/SevenOutman/stylelint-15-import-notation-less-repro) to demo a minimal repro.

Import an `a.less` in a `b.less` with `@import` at-rule.

```less
@import "a"
```

### What minimal configuration is needed to reproduce the bug?

```javascript
module.exports = {
  extends: ["stylelint-config-standard"],
  customSyntax: "postcss-less",
};
```

### How did you run Stylelint?

```shell
stylelint *.less
```

### Which Stylelint-related dependencies are you using?

```json
{
  "stylelint": "^15.10.2",
  "stylelint-config-standard": "^34.0.0"
}
```

### What did you expect to happen?

`import-notation` error should not be reported as it's valid import usage per [Less docs](https://lesscss.org/features/#import-atrules-feature)

### What actually happened?

The following problem is reported:

```

b.less
 1:9  ✖  Expected ""a"" to be "url("a")"  import-notation

1 problem (1 error, 0 warnings)

```

### Do you have a proposal to fix the bug?

Add built-in support to disable this rule in Less files, or add an option for this rule to ignore specific files.

**Comment by ybiquitous** [comment]
@SevenOutman Thanks for the report with a minimal reproduction repository. I also can reproduce it with [this demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIACBLAtgBwgJwC4AIAdEAQ2IG4QAaEAMxQBsYA5EtORGADzYyYDomAZyHVwEAHb0A5ghDBCE-ERDdcMCVCHF4+ANqLly4kNwBPJgxQTcAWkhSU026ZKaS2KMUP4AulR9iMABXUwg0AGUzGxIuHRUsUzARW2FtEEUAXzEHGQAxHDQSXDkAKyFJMVgMUUQFJRVTCxgrG3jiAEYAVn4OgAZ+ACZiAIaTc0trO1ynF1w3KA8vBBUAZgAWfgG+kcCQRNxkoVSYEXaQADYtre8JbMygA).

But I have a question. Can you specify the [primary option `"string"`](https://stylelint.io/user-guide/rules/import-notation/#string) as below?

```json
{
  "extends": ["stylelint-config-standard"],
  "customSyntax": "postcss-less",
  "rules": {
    "import-notation": "string"
  }
}
```

**Comment by SevenOutman** [comment]
> Can you specify the [primary option `"string"`](https://stylelint.io/user-guide/rules/import-notation/#string) as below?
> 
> ```json
> {
>   "extends": ["stylelint-config-standard"],
>   "customSyntax": "postcss-less",
>   "rules": {
>     "import-notation": "string"
>   }
> }
> ```

Ah, yes it works. How could I missed that. Would this be considered the recommended way to deal with this rule in projects with Less?

BTW the playground is cool.

**Comment by ybiquitous** [comment]
> Would this be considered the recommended way to deal with this rule in projects with Less?

I'm unfamiliar with Less, so I don't know whether `"import-notation": "string"` is recommended for Less projects. But overriding a sharable config is usually performed for specific projects.

For your reference, [`stylelint-config-standard-less`](https://www.npmjs.com/package/stylelint-config-standard-less), which is provided by a community, doesn't include `import-notation`.

> BTW the playground is cool.

Thanks :)

**Comment by jeddy3** [comment]
@SevenOutman Thanks for the report. As @ybiquitous said, you'll want to extend a community config if you're using Less (see our ["Getting started" guide](https://stylelint.io/user-guide/get-started)). The built-in rules and our official configs are for vanilla CSS.

> For your reference, [stylelint-config-standard-less](https://www.npmjs.com/package/stylelint-config-standard-less), which is provided by a community, doesn't include import-notation.

@SevenOutman You may want to open an issue in their tracker to turn the rule on using `"import-notation": "string"`, if that fits Less.

Closing the issue as the rule itself and our standard config as behaving as intended.

**Comment by SevenOutman** [comment]
Thanks. I'm going to try out [stylelint-config-standard-less](https://www.npmjs.com/package/stylelint-config-standard-less).
